SU2_CFD/src/fluid/CFluidModel.cpp

```diff
@@ -61,6 +65,8 @@ void CFluidModel::SetLaminarViscosityModel(const CConfig* config) {
       SU2_MPI::Error("Viscosity model not available.", CURRENT_FUNCTION);
       break;
   }
+//  }
+  // else {} 
```

Comment:
Now that you have this nice 'diff-view' you might want to go over it a bit and get rid of this commented out code, unnecessarily added lines and white spaces whenever you know that its superfluous. That helps keeping the diff smaller and bit more focussed on your work. Also splitting overlong lines at suitable points is a good idea here and there.

I think that kind of cleanup work is not too hard but will help you having less work down the road.